README.md

```diff
@@ -27,7 +27,7 @@ Python. When used inside a browser, Python has full access to the Web APIs.
 ## Try Pyodide (no installation needed)
 
 Try Pyodide in a
-[REPL](https://pyodide-cdn2.iodide.io/stable/full/console.html) directly in
+[REPL](https://pyodide.org/en/stable/console.html) directly in
```

Comment:
Previous link was incorrect. This one will work once release CI finishes building.